### PR TITLE
chore: update Mock Service Worker script

### PR DESCRIPTION
**Pull request by dzhavat**
### Workspace
- update Mock Service Worker script to get rid of console error

## References

- N/A

### CODE REVIEW

**Review by ManBearTM — DISMISSED** [review]
Looks good, I assume the code was generated? 🙂

**Comment by dzhavat** [comment]
> Looks good, I assume the code was generated? 🙂

Yes, it was generated. Looks like it's based on the `msw` version in `package.json`

File: package.json [hunk 1]
```diff
@@ -164,5 +164,8 @@
     "node": "16.16.0",
     "//yarn-comment": "Keep this in sync with GitHub actions pipelines",
     "yarn": "1.22.19"
+  },
+  "msw": {
+    "workerDirectory": "libs\\dh\\shared\\data-access-msw\\src\\assets"
```
Comment (LayZeeDK) [inline package.json:169]: suggestion: forward slash paths works across platforms
```suggestion
    "workerDirectory": "libs/dh/shared/data-access-msw/src/assets"

```

**Review by LayZeeDK — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (dzhavat) [reply package.json]: Thanks for your help Lars 🙏

**Comment by dzhavat** [comment]
Something's wrong with the "license check" and "format" actions. Closing this PR in favor of #864

**Comment by dzhavat** [comment]
@LayZeeDK thanks again for your contribution. It looks like by using forward slashes, we ended up in a weird state where MSW would generate a new mock Service Worker that is picked up by our "license header" and "format" steps that would begin to work against other. The double backslash was suggested by MSW itself, so we'll go with it for now until we find out what was wrong with the forward slash.

**Pull request closed without merging**